.pipelines/templates/ToolchainBuild.yml

```diff
@@ -11,6 +11,13 @@ parameters:
     default: ""
     displayName: "Daily build ID"
 
+  - name: isDailyBuildUpdateManifests
+    type: string
+    default: "true"
+    values:
+      - "true"
```

Comment:
Please also add the "default" option (`default: "default"`) like other string parameters pretending to be booleans. By forcing a specific value right now we're overwriting the toolkit's defaults, so if they ever change, we don't follow them. Thus the additional "default" string value.

Reply:
Updated